component-classes/index.js

```diff
@@ -156,19 +156,18 @@ export const toast = {
 };
 
 export const tabs = {
-  tabContainer: 'inline-grid relative',
+  tabContainer: 'inline-grid relative -mb-1',
   wunderbar: 'absolute s-border-selected -bottom-0 border-b-4 transition-all',
-  wrapperUnderlined: 'border-b border-transparent -mx-16 sm:mx-0 px-4 sm:px-0 mb-32 ',
+  wrapperUnderlined: 'inline-block border-b s-border mb-32',
```

Comment:
Setting `inline-block` class on the wrapper ensures the border doesn't bleed out, but reflects the Tabs actual width.